api/v1alpha1/proxyplugin_types.go

```diff
@@ -0,0 +1,88 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+const (
+	OpenShiftRouteEndpointType = ProxyPluginEndpointType("OpenShiftRouteEndpointType")
+)
+
+type ProxyPluginEndpointType string
+
+type ProxyPluginEntry struct {
+	// PluginName identifies the specific plugin, and is the next segment after '/plugin' in the request
```

Comment:
I think we should be using `.../plugins/<pluginName>/...` instead. Like `/workspaces/<workspaceName>/...` or `/namespaces/<namespaceName>/...`
```suggestion
	// PluginName identifies the specific plugin, and is the next segment after '/plugins/' in the request: '/plugins/<pluginName>/...'
```

Reply:
yeah I tried to subset the url explanation and as a result this comment is unclear

I'm going to update the godoc comment to more directly quote what is in the feature refinement doc for tekton results https://docs.google.com/document/d/1_VanTqldsChZXhUEG7n4gMyKiOL_lGGTNNIQM6xe6Ag/edit#heading=h.nhnim9xopxlz

That we support these 2 forms of URLs:
```
https://<proxy-host>/plugins/tekton-results/v1alpha2/<namespace-name>/<result-api> 
or 
https://<proxy-host>/plugins/tekton-results/workspaces/<workspace-name>/v1alpha2/<namespace-name>
```

and that `PluginName` corresponds to `tekton-results` in those 2 examples.

Of course, if you happen to see this @alexeykazakov before I update the PR and have further comment, I'll try to factor that in before my next push to the PR.

thanks